canonical-web-and-design/charmhub.io
# 1821 add args to library functions

## Done
- _Display function arguments on library docstring page._

## How to QA
- Run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/fran-wordpress/libraries/test
- See that functions have arguments displayed inside the brackets if they exist

_Will only work once https://github.com/canonical-web-and-design/canonicalwebteam.docstring-extractor/pull/8 is merged_

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1821

## Screenshots
![image](https://user-images.githubusercontent.com/40214246/109780410-e6dbb480-7bfe-11eb-8642-2d05b96f681a.png)
